source/index.html.erb

```diff
@@ -1,27 +1,29 @@
-<main class="index">
-    <h1><%= t("index.hello") %></h1>
-    <h3><%= t("index.welcome") %></h3>
-    <section class="index__cards">
-        <% link_to "mobile-heartbeat.html", :class => "index__case" do %>
-            <div class="index__case--overlay">
-                <p class="index__case-description"><%= t("index.mh-description") %></p>
-            </div>
-            <h4 class="index__case-heading"><%= t("index.mh") %></h4>
-            <%= image_tag "mh-thumb.png", :class => "index__thumbnail", :alt => t("index.mh-alt") %>
-        <% end %>
-        <% link_to "hmh-marketplace.html", :class => "index__case" do %>
-            <div class="index__case--overlay">
-                <p class="index__case-description"><%= t("index.hmh-description") %></p>
-            </div>
-            <h4 class="index__case-heading"><%= t("index.hmh") %></h4>
-            <%= image_tag "hmh-thumb.png", :class => "index__thumbnail", :alt => t("index.hmh-alt") %>
-        <% end %>
-        <% link_to "pegasystems.html", :class => "index__case" do %>
-            <div class="index__case--overlay">
-                <p class="index__case-description"><%= t("index.pega-description") %></p>
-            </div>
-            <h4 class="index__case-heading"><%= t("index.pega") %></h4>
-            <%= image_tag "hmh-thumb.png", :class => "index__thumbnail", :alt => t("index.pega-alt") %>
-        <% end %>
-    </section>
-</main>
+---
+title: Jeff Stolz | Product Designer
```

Comment:
A funny thing I’ve noticed while using VoiceOver (and maybe other screen readers do the same), is that it will read “|” as “vertical pipe”…so when it reads the title of this page, it will read “Jeff Stolz vertical pipe product designer.” 

Switching to something like a hyphen (`-`) could be worthwhile.

Reply:
Hahaha what?? TIL. That's so interesting.

Reply:
But... I'm REALLY good at making vertical pipes.